jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -37,6 +37,7 @@ If you already have a `circle.yml` file, this article will help you make a copy
          docker:
            - image: circleci/ruby:2.3
      ```
+     This first image in the list runs on the primary container where your build is run.
```

Comment:
Suggest "The primary container is an instance of the first list image listed. Your build commands run in this container."